Change _hfstorage to hfstorage

ankitageorge opened the pull request:
Summary: Change HF classes to not have an underscore, there-by making them public, we will add documentation to them following this

Test Plan:
ensure existing tests pass

Rollback Plan:

Differential Revision: D76364024


cc @H-Huang @awgu @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k

saumishr reviewed (approved):
Lets update our docs after landing this change.

ankitageorge committed:
Change _hfstorage to hfstorage (#155837)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/155837

Change HF classes to not have an underscore, there-by making them public, we will add documentation to them following this

Test Plan:
ensure existing tests pass

Rollback Plan:

Reviewed By: saumishr

Differential Revision: D76364024

ankitageorge commented:
@pytorchmergebot merge
